wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 ## Jamulus Software
 
-### Use of Profile Information
+Jamulus is open source software and can therefore be modified by others. As such, the Jamulus project makes no representations related to privacy, data collection, or security with respect to your use of the software.  
 
-When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it.  Note that, depending on the other client, you may not see them.  Also, the Server operator can see the name you set in My Profile and your IP address, using the Server either through the GUI or JSON-RPC.
+## General Information
 
-When you connect to a Server, your profile may also be available to third parties from the Directory to which that Server is registered. This can be for informational purposes about the status of the public Jamulus network (for example, [here](https://explorer.jamulus.io/)), but may not be limited to that. Profile information is not otherwise logged or stored by the Jamulus Server you are connected to, or by the Jamulus Directory, but may be stored or processed by third parties.
+Under normal use with unmodified software, your user profile information is exchanged with Servers you connect to, peers connected to those same Servers, and to any interested third party (including Directories) that uses the Jamulus protocol.  This information is limited to your Jamulus name, city, country, instrument, and skill level as you have set them in your profile.  The Servers you connect to will also have access to your internet address (IP Address) as it is required for the software to work; but this information is not shared with peers on the same Server or available to third parties. 
 
-### Use of IP Addresses
+All communications between and among Clients, Servers, Directories, and third-party protocol users are sent without encryption.
 
-When you connect to Server, the Server operator can see your IP address while you are connected.  If the Server operator has enabled logging (which is off by default) your IP address will also be logged and stored in the Server's log file.
+## Jamulus Servers
 
-The IP addresses of all Servers registered with the Directory can also be seen by third parties for informational or other purposes (for example [here](https://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
+When you connect to a Server, either directly or through a Directory, the operator of that Server is responsible for its operation policy, privacy policy, and data use policy. While unmodified Servers do not log or store your connection or profile information, some modified Servers may do this; therefore, you should have no expectation of privacy with respect to your profile information or internet address.
```

Comment:
Store your connection or profile information ... by default ...


IP's are logged if you use the .deb file as the Server outputs it to stdout and this is logged usually.

Reply:
Sounds like our Server emits IP addresses to stdout. Does our code (including scripts we provide) also log this? (Like bash with > I would assume.) If not, then our unmodified software is not logging.

Reply:
I think the line "you should have no expectation of privacy with respect to your profile information or internet address" should be clear enough :-) (I think @ann0see means the .deb has a [systemd unit file](https://github.com/jamulussoftware/jamulus/blob/master/linux/debian/jamulus-headless.service) distributed with it that logs IP addresses to journald, which you could turn off).

Reply:
Exactly what @gilgongo said.

Reply:
> While unmodified Servers do not log or store your connection or profile information

This simply is _not true_.  The _unmodified software_ is perfectly capable of doing this.

Reply:
Updated to say that now.